app/serializers/rest/instance_serializer.rb

```diff
@@ -99,6 +99,8 @@ def configuration
       translation: {
         enabled: TranslationService.configured?,
       },
+
+      closed_federation: limited_federation?,
```

Comment:
Any reason not to call this `limited_federation`? I feel like I have never seen the term `closed_federation` and it also does not describe the situation very well imho (as such servers can and probably will federate at least with some servers).

Reply:
I would suggest `silo` as well...

Reply:
I would use either `limited_federation` (consistency with the docs / config variable), or `restricted_federation`?

Reply:
Changed to `limited_federation`